fc/jira/fcissue.py

```diff
@@ -45,3 +45,12 @@ def _transition(self, id_of_transition: int):
 
     def _get_transition_dict(self) -> dict:
         raise NotImplementedError
+
+    def update_vfr(self, duration: int, cost_of_delay: int) -> float:
+        raise NotImplementedError
```

Comment:
This method is no longer needed, correct?  So we can delete it.

Reply:
Ach yes, and i meant to remove it.